target/linux/layerscape/image/gen_sdcard_head_img.sh

```diff
@@ -7,16 +7,18 @@
 #
 
 set -x
-[ $# -eq 3 ] || {
-    echo "SYNTAX: $0 <file> <rootfs part offset> <rootfs size>"
+[ $# -eq 5 ] || {
+    echo "SYNTAX: $0 <file> <kernel part offset> <kernel size> <kernel part offset> <rootfs size>"
```

Comment:
fourth argument should be "rootfs part offset"

Reply:
Done.